docs/guided-tour.md

```diff
@@ -56,29 +56,45 @@ During this tutorial, you're going to try out some cool features of Sandstorm. Y
 
 One of the common reasons to use Sandstorm is to have online collaboration software that runs on your own server.
 
-Etherpad is a real-time text editor that demonstrates how to use Sandstorm that way. Make sure you're logged into Sandstorm (if using the demo, click **Start the demo**). Then click:
+Etherpad is a real-time text editor that demonstrates how to use Sandstorm that way. Make sure you're logged into Sandstorm (if using the demo, click **Try a quick demo**). Then:
 
-* **Install apps** - this takes you to the Sandstorm app list. (Soon to be the Sandstorm App Marketplace!)
-* Click the **Install** button next to _Etherpad_. (Sandstorm may ask you for confirmation.)
-* Click **New Etherpad Document** button on your Sandstorm.
+* Click **Install...** - this takes you to the Sandstorm App Market.
+* Find _Etherpad_ and click the **Install** button. Sandstorm will ask you for confirmation.
+* Click **Install Etherpad**.
+* Once Etherpad is installed, click **Create new pad**.
 
 You're now in an Etherpad document. Installing Etherpad allows you to create as many Etherpad documents as you want.
 
-If you want to create spreadsheets on your server, consider installing _Ethercalc_.
+If you want to create spreadsheets on your server, consider installing [_Ethercalc_](https://apps.sandstorm.io/app/a0n6hwm32zjsrzes8gnjg734dh6jwt7x83xdgytspe761pe2asw0).
 
-If you want to edit scientific documents, consider installing _ShareLaTeX_.
+If you want to edit scientific documents, consider installing [_ShareLaTeX_](https://apps.sandstorm.io/app/5vuv7v0w7gu20z72m78n83rx9qqtqpmtk32f39823wh967z226qh).
 
-If you want to create presentations, consider installing _Hacker Slides_.
+If you want to create presentations, consider installing [_Hacker Slides_](https://apps.sandstorm.io/app/7qvcjh7gk0rzdx1s3c8gufd288sesf6vvdt297756xcv4q8xxvhh).
+
+If you want to organize tasks with a kanban board, consider installing [_Wekan_](https://apps.sandstorm.io/app/m86q05rdvj14yvn78ghaxynqz7u2svw6rnttptxx49g1785cdv1h).
 
 Each document is an _instance_of that app, with Sandstorm isolating each one from each other.
 
 ### Sharing
 
 In Sandstorm, each app instance is private by default. In order to collaborate, you create a _sharing link_.
 
-In your Etherpad document, look for the blue **Share** link at the top (or the side) of the document. Click it, choose a permission level, and click **create**.
+In your Etherpad document, look for the **Share Access** icon and link in the black topbar. You can share this grain by sending an email invite or getting a shareable link.
```

Comment:
I figured I'd include both options in the guide since they're there. Also email invitations display first under "Share Access" by default. I don't know if we can make an assumption as to what people prefer but if you think using a sharing link is a better experience then we can just explain that one.